Ladybird/Qt/InspectorWidget.cpp

```diff
@@ -54,10 +60,15 @@ InspectorWidget::InspectorWidget(QWidget* tab, WebContentView& content_view)
     m_dom_node_text_context_menu->addSeparator();
     m_dom_node_text_context_menu->addAction(m_delete_node_action);
 
+    auto* create_child_menu = new QMenu("Create child", this);
+    create_child_menu->addAction(m_create_child_element_action);
+    create_child_menu->addAction(m_create_child_text_node_action);
```

Comment:
At some point maybe it'll make sense to generate these context menus at runtime from a platform-agnostic set of menu items with shortcuts, callbacks, etc?

Reply:
Yeah I had a similar thought the other day looking at the context menus in Qt/Tab.cpp - there's quite a bit of code dedicated to just setting those up. Would be a nice win to be able to generate those from LibWebView data.